src/button/buttonview.js

```diff
@@ -22,20 +22,34 @@ export default class ButtonView extends View {
 
 		const bind = this.bind;
 
+		// Set default value for `noText` property.
+		model.set( 'noText', !!model.noText );
```

Comment:
The component should not change the model it obtains. In fact, this line is not needed as `bind.to()` for an attribute which is not yet in the model will also work if the attribute is eventually set.